api/v1alpha1/loadbalancer_types.go

```diff
@@ -134,6 +144,45 @@ type Cookie struct {
 	Attributes map[string]string `json:"attributes,omitempty"`
 }
 
+// ClientSideWeightedRoundRobin defines configuration for Envoy's Client-Side Weighted Round Robin policy.
+// See Envoy proto: envoy.extensions.load_balancing_policies.client_side_weighted_round_robin.v3.ClientSideWeightedRoundRobin
+// Note: SlowStart is not supported for this policy in Envoy Gateway at this time.
+type ClientSideWeightedRoundRobin struct {
+	// Whether to enable out-of-band utilization reporting collection from the endpoints.
+	// By default, per-request utilization reporting is used.
+	// +optional
+	EnableOOBLoadReport *bool `json:"enableOOBLoadReport,omitempty"`
```

Comment:
OOB is not a common short name, prefer to use full name here.

cc @envoyproxy/gateway-maintainers